game-core/src/main/java/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -115,18 +116,14 @@ public void draw(
         uiContext.getUnitImageFactory().getImage(type, owner, damagedImage, disabled);
 
     if (img.isEmpty() && !uiContext.isShutDown()) {
-      final String imageQualifier;
```

Comment:
It's really cool to see this code be removed  : )
To boot, the error messaging will be much more informative.